setup.py

```diff
@@ -19,14 +19,13 @@
     url="https://github.com/mintapi/mintapi",
     install_requires=[
         "configargparse",
-        "future",
-        "mock",
```

Comment:
These ones aren't actually used?

Reply:
They're included in the standard library of python 3.

Reply:
Or rather, mock is. Future is no longer used as it was for python 2 to 3 compatibility.